packages/block-library/src/paragraph/edit.js

```diff
@@ -57,7 +57,7 @@ function ParagraphRTLToolbar( { direction, setDirection } ) {
 }
 
 function useDropCap( isDropCap, fontSize, styleFontSize ) {
-	const isDisabled = useEditorFeature( '__experimentalDisableDropCap' );
+	const isDisabled = ! useEditorFeature( 'typography.dropCap' );
```

Comment:
Would it make sense to use [ 'typography', 'dropCap' ] directly like we force lodash paths to be using lint rules?

Reply:
I would prefer to avoid optimizing the hook based on the internal implementation that can change over time.